torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -1142,75 +1144,98 @@ std::pair<NvrtcFunction, std::string> nvrtcCompile(
   at::globalContext().getNVRTC().nvrtcAddNameExpression(
       program, func_name.c_str());
 
-  {
-    FUSER_PERF_SCOPE("executor_utils::Nvrtc::CompileProgram");
+  // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
+  std::vector<char> ptx;
```

Comment:
Might be worthwhile to factor out https://github.com/csarofeen/pytorch/blob/devel/torch/csrc/jit/codegen/cuda/codegen.cpp#L31-L80 and reuse it here with a different string separator.